index.html

```diff
@@ -0,0 +1,142 @@
+				 <a class="links" href="#">News</a>
+				 <div class="desktop-nav-btn">
+					 <div class="desktop-btn">ZTM Course</div>
+				 </div>
+				</div>
+			 </nav>
+			 <nav class="mobile-nav">
+	      <a id="onclick-open" href="javascript:void(0);" >
+			  <img class="menu-icon" src="imgs/menuIcon.svg" alt="menu icon">
+			</a>
+		  <div class="mobile-menu">
+			<a id="onclick-close" href="javascript:void(0);">
+				<img class="close-btn" id="closeImg" src="imgs/close.svg" alt="closeButton">
+			  </a>
+		 <a id="onclick-about" href="about.html">About</a>
+		 <a id="onclick-program" href="#">Program</a>
+		 <a id="onclick-join" href="#">Join</a>
+		 <a id="onclick-partners" href="#">Partners</a>
+		 <a id="onclick-news" href="#">News</a>
+		</div>
+			</nav>
+
+<div class="container-content">
+	<h1>"Zero to Mastery!"</h1>
+	<h2>THE COMPLETE WEB DEVELOPER IN 2022: ZERO TO MASTERY</h2>
+</div>
+	<div class="container-descreption">
+		<p>Skills that will allow you to apply for jobs like: Web Developer, Software Developer, Front End Developer, Javascript Developer, and Full Stack Developer</p>
+			<div class="descreption">
+				<h3 class="update">Last updated 1/2022</h3>
+				<h3 class="language">English</h3>
+			</div>
+			<div  class="subtitles">
+				<h3>Subtitles: 7 different languages</h3>
+			</div>
+	</div>
+</div>
+ </header>
+ <section id="main" class="main">
+   <a class="link" href="#main">Main Program</a>
+	<ul class="skills">
+	  <li class="tools">
+			<div class="left">
+			  <img src="imgs/htmlcon.svg" alt="htmlIcon">
+				<h4>HTML</h4>
+			</div>
+			<div class="right">
+				<p>HTML helps you build your own unique website.</p>
+			</div>
+		</li>
+		<li class="tools">
+			<div class="left">
+				<img src="imgs/cssIcon.svg" alt="cssIcon">
+				<h4>CSS</h4>
+			</div>
+			<div class="right">
+				<p>CSS styles your content semantically.</p>
+			</div> 
+		</li>
+		<li class="tools">
+			<div class="left">
+				<img src="imgs/JSicon.svg" alt="JSIcon">
+				<h4>JS</h4>
+			</div>
+			<div class="right">
+				<p>JavaScript controls the dynamic elements of web pages.</p>
+			</div> 
+		</li>
+		<li class="tools">
+			<div class="left">
+				<img class="react" src="imgs/reactIcon.svg" alt="htmlIcon">
+				<h4>REACT</h4>
+			</div>
+			<div class="right">
+				<p>React is a JavaScript library (not a framework) that creates user interfaces (UIs).</p>
+			</div> 
+		</li>
+		<li class="tools">
+			<div class="left">
+				<img class="node" src="imgs/node.svg" alt="htmlIcon">
+				<h4>NODE JS</h4>
+			</div>
+			<div class="right">	
+			<p>is an open-source backend javascript runtime environment.</p>
+			</div> 
+		</li>		
+	</ul>
+		<div class="join-btn">
+			<button type="button">Join the ZTR Course Now!</button>
+		</div>
+ </section>
+ <section id="instructors" class="instructors">
+	 <a class="link2" href="#instructors"><h6>Instructors</h6></a>
+	 <ul class="instructorsList">
+
+	 </ul>
+ </section>
```

Comment:
- [ ] I'd suggest you make sure the instructor section articles are organized in two columns of three articles each as seen in the original design.😎
- Your design | Original design
![screenshot (11)](https://user-images.githubusercontent.com/44624138/152585477-3a6bfa67-d7b2-4d57-99d0-96309514b785.png) | ![screenshot (10)](https://user-images.githubusercontent.com/44624138/152585308-e32f1121-5bcb-4798-a3a9-681f1b9f658b.png)